test/test_ast_generation.py

```diff
@@ -37,6 +37,8 @@ def parser_extension(tmp_path_factory: Any) -> Any:
 
 @pytest.mark.parametrize("filename", PYTHON_SOURCE_FILENAMES)
 def test_ast_generation_on_source_files(parser_extension: Any, filename: PurePath) -> None:
+    if filename == "group.py":
```

Comment:
Suggestion: We can write an independent test for this additionally, and mark it for now as `xfail` so we don't forget and know when is fixed